packages/gp2-components/src/organisms/ProjectsHeader.tsx

```diff
@@ -1,14 +1,27 @@
+import { Link, externalLinkIcon } from '@asap-hub/react-components';
 import { projectsImage } from '../images';
 import PageBanner from './PageBanner';
 
 const props = {
   image: projectsImage,
   position: 'center',
-  title: 'Project Directory',
+  title: 'Projects',
   description:
-    'Explore past and present GP2 projects and discover open opportunities to join them.',
+    'Explore GP2 projects being carried out by the network. If you are interested in starting a new project, please request a new project below.',
 };
 
-const ProjectsHeader: React.FC = () => <PageBanner {...props} />;
+const ProjectsHeader: React.FC = () => (
+  <PageBanner {...props}>
+    <Link
+      href="https://docs.google.com/forms/d/e/1FAIpQLScYnKgzk-gxFW6a8CgEkwowjCnWGdWqLxwF9YWacYHMnSaPzg/viewform"
+      label="Request New Project"
+      primary
+      buttonStyle
```

Comment:
The padding is huge (see below). I think adding the prop `small` would do it.
![Screenshot 2023-07-07 at 12 24 06](https://github.com/yldio/asap-hub/assets/37268411/0d6e8df9-ac39-4cbf-929b-ff068022f06c)

Figma:
![Screenshot 2023-07-07 at 12 25 21](https://github.com/yldio/asap-hub/assets/37268411/6a4b3718-7006-4b78-a35c-6cb8df6eea55)